commune_app/models.py

```diff
@@ -1,3 +1,16 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class Decision(models.Model):
```

Comment:
@NicolePauzner I think it will be better to create separate Django app for each module in this way you will be able to work on each module with fewer conflicts with the rest of the team